src/Microsoft.CodeAnalysis.Razor/ViewComponentTypes.cs

```diff
@@ -21,6 +21,8 @@ internal static class ViewComponentTypes
 
         public const string Task = "System.Threading.Tasks.Task";
 
+        public const string IDictionary = "System.Collections.Generic.IDictionary`2";
```

Comment:
Why put this here and not leave on the `TagHelperTypes`? The `IDictionary` support comes from the TagHelper system. The ViewCOmponentTagHelper system just detects that something is an idictionary.